module/zfs/dbuf.c

```diff
@@ -302,6 +303,12 @@ dbuf_hash(void *os, uint64_t obj, uint8_t lvl, uint64_t blkid)
 	return (cityhash4((uintptr_t)os, obj, (uint64_t)lvl, blkid));
 }
 
+#define	DBUF_STATE_CHANGE(db, op, state, why) do {		\
```

Comment:
This PR seems like it includes sections that are in #9923?  This is the original macro I was referring to.

Reply:
@wca It hasn’t been rebased.